api/blast_ui.api.inc

```diff
@@ -1,12 +1,96 @@
 <?php
-
 /**
  * @file
  * Contains more generally applicable functions as well as some meant to help developers
  * Plug-in to the BLAST UI functionality
  */
 
 /**
+ * This program is used for services module specifically.
+ * Run BLAST (should be called from the command-line)
+ *
+ * @param $program
+ *   Which BLAST program to run (ie: 'blastn', 'tblastn', tblastx', 'blastp','blastx')
+ * @param $query
+ *   The full path and filename of the query FASTA file
+ * @param $database
+ *   The full path and filename prefix (excluding .nhr, .nin, .nsq, etc.)
+ * @param $output_filestub
+ *   The filename (not including path) to give the results. Should not include file type suffix
+ * @param $options
+ *   An array of additional option where the key is the name of the option used by
+ *   BLAST (ie: 'num_alignments') and the value is relates to this particular
+ *   BLAST job (ie: 250)
+ * @param $format 
+ *   Which specifies Json or asn format of file or program
+ * @param  $format_type 
+ * which specify the number of format used by BLAST e.g json has 11 and json as 12. 
```

Comment:
Documentation for $format and $format_type needs to be improved. Are you expecting $format = 'JSON' or $format = 12? Same goes for $format_type. Also, you say that json is both 11 and 12 (assuming this is a typo).